### PR TITLE
chainparams: Bump assumed valid hash

### PR DESCRIPTION
**Pull request by maflcko**
See https://github.com/bitcoin/bitcoin/blob/master/doc/release-process.md#before-branch-off

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -100,8 +100,8 @@ class CMainParams : public CChainParams {
         pchMessageStart[3] = 0xd9;
         nDefaultPort = 8333;
         nPruneAfterHeight = 100000;
-        m_assumed_blockchain_size = 280;
-        m_assumed_chain_state_size = 4;
+        m_assumed_blockchain_size = 320;
+        m_assumed_chain_state_size = 6;
```
Comment (sipa) [inline src/chainparams.cpp:104]: My chainstate/ dir is 3.8 GB. Are you increasing this to give some room for expansion, or seeing different numbers than I do?

File: src/chainparams.cpp [hunk 2]
```diff
@@ -84,10 +84,10 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nTimeout = 1230767999; // December 31, 2008
 
         // The best chain should have at least this much work.
-        consensus.nMinimumChainWork = uint256S("0x000000000000000000000000000000000000000008ea3cf107ae0dec57f03fe8");
+        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000e1ab5ec9348e9f4b8eb8154");
 
         // By default assume that the signatures in ancestors of this block are valid.
-        consensus.defaultAssumeValid = uint256S("0x00000000000000000005f8920febd3925f8272a6a71237563d78c2edfdd09ddf"); // 597379
+        consensus.defaultAssumeValid = uint256S("0x0000000000000000000f2adce67e49b0b6bdeb9de8b7c3d7e93b21e7fc1e819d"); // 623950
```
Comment (instagibbs) [inline src/chainparams.cpp:90]: ACK hash, height on personal node

File: src/chainparams.cpp (hunk 1 above)
Comment (MarcoFalke) [reply src/chainparams.cpp:104]: How do you check the size of the chainstate dir?

```
$ du -sh ~/Downloads/bm/chainstate/
4.6G	/home/marco/Downloads/bm/chainstate/

Comment (instagibbs) [reply src/chainparams.cpp:104]:
```
du -sh ~/.bitcoin/chainstate/
3.8G	/home/instagibbs/.bitcoin/chainstate/
```

**Review by fanquake — APPROVED** [review]
ACK 222253e37d9702181fe754d776177962d6794f30 - matches output on my node. Also have 3.8GB for the chainstate dir. If anyone wants to know what commands they should run / what to check; I have some docs on [assumevalid updates](https://github.com/fanquake/core-review/blob/master/update-assumevalid.md).

**Review by elichai — APPROVED** [review]
ACK 222253e37d9702181fe754d776177962d6794f30
All the mainnet params match my local node.
Have 3.7GB chainstate dir, and 287GB blocks dir.

**Comment by MarcoFalke** [comment]
Open-Close to re-run ci. See https://github.com/bitcoin/bitcoin/issues/15847#issuecomment-487591130

Previous runs:
* https://ci.appveyor.com/project/DrahtBot/bitcoin/builds/31891367
* https://travis-ci.org/github/bitcoin/bitcoin/builds/669896548

**Review by theStack — APPROVED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/18500/commits/222253e37d9702181fe754d776177962d6794f30
Verified the parameters on my node with:
```
$ bitcoin-cli getblockstats 623950
$ bitcoin-cli getblock 0000000000000000000f2adce67e49b0b6bdeb9de8b7c3d7e93b21e7fc1e819d
$ bitcoin-cli getchaintxstats 4096 0000000000000000000f2adce67e49b0b6bdeb9de8b7c3d7e93b21e7fc1e819d
```
The same procedure also for the testnet.
Mainnet chainstate dir has 3.8GB size, blocks dir has 287GB.
Testnet chainstate dir has 16KB size, blocks dir has 26GB.

File: src/chainparams.cpp [hunk 3]
```diff
@@ -156,10 +156,10 @@ class CMainParams : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            // Data from RPC: getchaintxstats 4096 00000000000000000005f8920febd3925f8272a6a71237563d78c2edfdd09ddf
-            /* nTime    */ 1569926786,
-            /* nTxCount */ 460596047,
-            /* dTxRate  */ 3.77848885073875,
+            // Data from RPC: getchaintxstats 4096 0000000000000000000f2adce67e49b0b6bdeb9de8b7c3d7e93b21e7fc1e819d
```
Comment (instagibbs) [inline src/chainparams.cpp:159]: what's 4096 in reference to here? isn't this a height argument?

**Review by instagibbs — APPROVED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/18500/commits/222253e37d9702181fe754d776177962d6794f30 all mainnet params, did not check testnet

**Pull request merged**